metadata.json

```diff
@@ -27,7 +27,7 @@
     },
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 1.0.0 < 3.0.0"
+      "version_requirement": ">= 1.0.0 <= 3.0.0"
```

Comment:
Probably best to make this `< 4.0.0` otherwise it'll need updating as soon as 3.0.1 is released.

Reply:
Will do!

Reply:
a 4.5.1 of puppet/archive???

Reply:
Ah no sorry man, I had the wrong tab opened (puppetlabs-apt) and quotes from there. messing this one up. I'll create a new one with < 4.0.0